app/src/main/java/de/danoeh/antennapod/adapter/StatisticsListAdapter.java

```diff
@@ -2,32 +2,27 @@
 
 import android.content.Context;
 import androidx.annotation.NonNull;
-import androidx.appcompat.app.AlertDialog;
 import androidx.recyclerview.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.ImageView;
 import android.widget.TextView;
-import com.bumptech.glide.Glide;
-import com.bumptech.glide.request.RequestOptions;
 import de.danoeh.antennapod.R;
-import de.danoeh.antennapod.core.glide.ApGlideSettings;
 import de.danoeh.antennapod.core.storage.DBReader;
-import de.danoeh.antennapod.core.util.Converter;
 import de.danoeh.antennapod.view.PieChartView;
 
 /**
- * Adapter for the statistics list
+ * Parent Adapter for the playback and download statistics list
  */
-public class StatisticsListAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
-    private static final int TYPE_HEADER = 0;
+public abstract class StatisticsListAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+    static final int TYPE_HEADER = 0;
     private static final int TYPE_FEED = 1;
-    private final Context context;
-    private DBReader.StatisticsData statisticsData;
-    private boolean countAll = true;
+    final Context context;
+    DBReader.StatisticsData statisticsData;
+    boolean countAll = true;
```

Comment:
Please move this to the child class. It's only relevant for the playback list.

Reply:
Now there is a copy in the child class. Please remove it here :)

Apart from that, I am happy. Works fine!